frontend/packages/pipelines-plugin/src/components/import/pipeline/pipeline-template-utils.ts

```diff
@@ -52,6 +53,8 @@ export const getPipelineParams = (
         return { ...param, default: getImageUrl(name, namespace) };
       case 'DOCKERFILE':
         return { ...param, default: dockerfilePath };
+      case 'VERSION':
```

Comment:
@karthikjeeyar  How difficult would it be to support both VERSION and the MAJOR/MINOR counterparts that are going to be removed in 1.4?

Reply:
@andrewballantyne  Setting the MAJOR & MINOR values is not a big deal, but the template selection process is tricky.
1.3 will have multiple templates for same builder image. eg: s2i-java-8 and s2i-java-9 so we need to fetch based on the annotations `pipeline.openshift.io/supported-version`.

In 1.4, there will be only one template per builder Image, so  when to use multiple fetching vs default fetching of templates is not straight forward.
On top of that all this needs to be removed when 1.4 is available. What are your thoughts on this ?

Reply:
Yeah.... I was worried that was going to be your answer. That was my thoughts too. Let's just trust upstream will come and deliver and we don't need to put in a fallback in case it doesn't land. I retract my request 🙂